samples/ControlCatalog.Android/MainActivity.cs

```diff
@@ -5,16 +5,8 @@
 
 namespace ControlCatalog.Android
 {
-    [Activity(Label = "ControlCatalog.Android", Theme = "@style/MyTheme.NoActionBar", Icon = "@drawable/icon", LaunchMode = LaunchMode.SingleInstance, ConfigurationChanges = ConfigChanges.Orientation | ConfigChanges.ScreenSize)]
```

Comment:
Should we keep SingleInstance? Otherwise Android might create multiple activities in certain scenarios.

Reply:
SingleInstance causes the OS to create a new instance of the app when it's launched from app launcher, as the main activity is not the launch activity in our case, the splach activity is. It becomes a different task. SingleTop or SingleTask will be suitable  here, with SIngleTop being recommended.